runs/ws_logs.py

```diff
@@ -0,0 +1,35 @@
+"""
+This module handles logs received via websockets.
+"""
+import json
+import time
+
+from .models import ContentChannelRun
+from channels.sessions import channel_session
+from django.http import Http404
+
+
+@channel_session
+def connect(message):
+    # Expected path format: /logs/<run_id>/
+    _, run_id = message['path'].strip('/').split('/')
```

Comment:
This feels like it could cause problems in the long run. 

Why not add the `run_id` as another json field along with the data?
See: https://github.com/learningequality/ricecooker/pull/79/files#r121937489